src/panels/config/integrations/ha-config-integration-page.ts

```diff
@@ -338,41 +340,72 @@ class HaConfigIntegrationPage extends SubscribeMixin(LitElement) {
                 ${this._manifest?.version != null
                   ? html`<div class="version">${this._manifest.version}</div>`
                   : nothing}
+                ${this._manifest?.quality_scale &&
+                this._manifest?.quality_scale !== "internal"
+                  ? html`
+                      <div class="quality-scale integration-info">
+                        <ha-svg-icon
+                          class=${`${this._manifest.quality_scale}-medal`}
+                          .path=${this._manifest.quality_scale === "platinum"
+                            ? mdiTrophy
+                            : mdiMedal}
+                        ></ha-svg-icon>
+                        <span>
+                          ${this.hass.localize(
+                            `ui.panel.config.integrations.config_entry.${this._manifest.quality_scale}_quality`,
+                            {
+                              quality_scale: html`
+                                <a
+                                  href=${documentationUrl(
+                                    this.hass,
+                                    `/docs/quality_scale/#${this._manifest.quality_scale}-`
+                                  )}
+                                  rel="noopener noreferrer"
+                                  target="_blank"
+                                >
+                                  ${this.hass.localize(
+                                    "ui.panel.config.integrations.config_entry.quality_scale"
+                                  )}
+                                </a>
+                              `,
+                            }
+                          )}
+                        </span>
+                      </div>
+                    `
+                  : nothing}
                 ${this._manifest?.is_built_in === false
-                  ? html`<ha-alert alert-type="warning"
-                      ><ha-svg-icon
-                        slot="icon"
+                  ? html`<div class="integration-info warn">
+                      <ha-svg-icon
+                        class="warning"
                         path=${mdiPackageVariant}
                       ></ha-svg-icon>
                       ${this.hass.localize(
                         "ui.panel.config.integrations.config_entry.custom_integration"
-                      )}</ha-alert
-                    >`
-                  : ""}
+                      )}
+                    </div>`
+                  : nothing}
                 ${this._manifest?.iot_class?.startsWith("cloud_")
-                  ? html`<ha-alert
-                      ><ha-svg-icon slot="icon" path=${mdiCloud}></ha-svg-icon
-                      >${this.hass.localize(
+                  ? html`<div class="integration-info">
+                      <ha-svg-icon .path=${mdiWeb}></ha-svg-icon>
```

Comment:
If we update the icon here, we should also update it on the integration page and the new integration picker

Reply:
That icon is already there. Just in integration detail it was a cloud.